lib/rules/prefer-explicit-assert.ts

```diff
@@ -34,12 +37,18 @@ export default ESLintUtils.RuleCreator(getDocsUrl)<Options, MessageIds>({
     messages: {
       preferExplicitAssert:
         'Wrap stand-alone `getBy*` query with `expect` function for better explicit assertion',
+      preferExplicitAssertAssertion:
+        '`getBy*` queries must be asserted with `{{assertion}}`',
     },
     fixable: null,
     schema: [
       {
         type: 'object',
+        additionalProperties: false,
```

Comment:
I originally didn't have to add the `assertion` type because the schema wasn't validating excess properties. Technically this could be considered a breaking change if any folks have an invalid configuration, but this is now consistent with other rules (eg: [`consistent-data-testid.`](https://github.com/testing-library/eslint-plugin-testing-library/blob/609c6dbe9e97eb36993ca1d5c05ad4bc2f66e405/lib/rules/consistent-data-testid.ts#L33)). Thought it was worthwhile to make this change to also catch typos (eg: `customQueryName`)?